validator/client/src/main/java/tech/pegasys/teku/validator/client/doppelganger/DoppelgangerDetectionShutDown.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright ConsenSys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.validator.client.doppelganger;
+
+import java.util.List;
+import tech.pegasys.teku.bls.BLSPublicKey;
+
+public class DoppelgangerDetectionShutDown implements DoppelgangerDetectionAction {
+  @Override
+  public void shutDown() {
+    System.exit(1);
```

Comment:
Do we need to log something to say why we're shutting down in this case?

Reply:
I added a log